docs/connect-field-devices/asset-implemetation.md

```diff
@@ -56,9 +56,7 @@ The Asset supports applying a scale and offset to the values obtained by the att
 The following **mode**s are implemented for computing scale and offset:
 
```

Comment:
IMO, the term **mode** here is a bit misleading. Is it better to present her the **scaleoffset.type** instead?

Reply:
with @nicolatimeus we decided to adopt this name, let see what he thinks about.